src/index.html

```diff
@@ -11,7 +11,194 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="fieldset firstBlock">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+            >
+        </label>
+
+        <label class="form-field">
+          Name:
+
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+
+          <input
+            type="date"
+            name="dateOfBirth"
+            required
+          >
+        </label>
+
+        <label class="form-field, last">
```

Comment:
multiple classes are separated with single space, without a comma.
```suggestion
        <label class="form-field last">
```

Reply:
done